logs_api/lib/opentelemetry/logs/logger.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Logs
+    # No-op implementation of logger.
+    class Logger
+      # Create a {LogRecord}
+      #
+      # @param timestamp [optional Float, Time] Time in nanoseconds since Unix
+      #   epoch when the event occurred measured by the origin clock, i.e. the
+      #   time at the source.
+      # @param observed_timestamp [optional Float, Time] Time in nanoseconds
+      #   since Unix epoch when the event was observed by the collection system.
+      #   Defaults to +Process.clock_gettime(Process::CLOCK_REALTIME)+.
+      # @param context [optional Context] The Context to associate with the
+      #   LogRecord. Defaults to +OpenTelemetry::Context.current+.
+      # @param severity_number [optional Integer] Numerical value of the
+      #   severity. Smaller numerical values correspond to less severe events
+      #   (such as debug events), larger numerical values correspond to more
+      #   severe events (such as errors and critical events).
+      # @param severity_text [optional String] Original string representation of
+      #   the severity as it is known at the source. Also known as log level.
+      # @param body [optional String, Numeric, Boolean, Array<String, Numeric,
+      #   Boolean>, Hash{String => String, Numeric, Boolean, Array<String,
+      #   Numeric, Boolean>}] A value containing the body of the log record.
+      # @param attributes [optional Hash{String => String, Numeric, Boolean,
+      #   Array<String, Numeric, Boolean>}] Additional information about the
+      #   event.
+      #
+      # @api public
+      def create_log_record(
```

Comment:
I notice the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#emit-a-logrecord) refers to "emitting" a log record.

I also noticed that [javascript](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/api-logs/src/types/Logger.ts#L25) calls this method `emit` and that it doesn't return a value. It might be a good idea to see what implementations look like in other languages. 

My initial impression is that this method creates a LogRecord internally and passes it on, without returning it to the caller.

Reply:
Thanks for bringing this up! 

[Python](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py#L100C5-L102), [PHP](https://github.com/open-telemetry/opentelemetry-php/blob/main/src/API/Logs/LoggerInterface.php#L9) and [Java](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/logs/DefaultLogger.java#L79) all have similar implementations to JavaScript. 

I've updated the method to be named `#emit` and return nil in 12d39578d21852bf1926a4b5f954fbb52af2cf9a